src/components/Reports/DonationsReport/DonationsReportTable.tsx

```diff
@@ -1,10 +1,242 @@
-import React from 'react';
-import { Box } from '@material-ui/core';
+import React, { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Button,
+  Link,
+  Divider,
+  styled,
+  Table,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import EditIcon from '@material-ui/icons/Edit';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { useTranslation } from 'react-i18next';
+import { DataGrid, GridColDef, GridCellParams } from '@material-ui/data-grid';
+import { DateTime } from 'luxon';
+
+import { EmptyDonationsTable } from '../../common/EmptyDonationsTable/EmptyDonationsTable';
+
+interface Donation {
+  date: Date;
+  partnerId: string;
+  partner: string;
+  currency: string;
+  foreignCurrency: string;
+  convertedAmount: number;
+  foreignAmount: number;
+  designation: string;
+  method: string;
+  id: string;
+}
 
 interface Props {
+  data: Donation[];
   accountListId: string;
 }
 
-export const DonationsReportTable: React.FC<Props> = () => {
-  return <Box />;
+const Data = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  '& .MuiDataGrid-row.Mui-even:not(:hover)': {
+    backgroundColor:
+      theme.palette.type === 'light'
+        ? theme.palette.common.white
+        : theme.palette.cruGrayLight.main,
+  },
+}));
+export const DonationsReportTable: React.FC<Props> = ({
+  data,
+  /*accountListId,*/
+}) => {
+  const { t } = useTranslation();
+  const accountCurrency = data[0].currency;
+  const link = (params: GridCellParams) => (
+    <Typography>
+      <Link href="contact.html">{params.value}</Link>
+    </Typography>
+  );
+  const amount = (params: GridCellParams) => {
+    const row = params.row as Donation;
+
+    return <Typography>{row.convertedAmount + ' ' + row.currency}</Typography>;
```

Comment:
Use JSX for string interpolation here
```suggestion
    return <Typography>{row.convertedAmount} {row.currency}</Typography>;
```

Between JSX and ES6 template literals, we shouldn't need to reach for ` + ` to concatenate strings anymore.